packages/components/src/config/variables.ts

```diff
@@ -152,3 +152,262 @@ export const ICONS: IconType[] = [
     'WINDOW_RESTORE_MAC',
     'TOR',
 ];
+
+export const FLAGS: FlagType[] = [
```

Comment:
can we somehow merge this with `packages/components/src/components/Flag/flags.ts` and have a single source of truth for the country codes / flags?

Reply:
That sounds reasonable. Do I see correctly that this export is used only in storybook and basically the same shortcuts are exported from the aformentioned file? In that case you may drop this export and use the other one just doing plain old `Object.keys().map`
Or maybe there is another reason to have it here I am missing?

Reply:
Done